src/pcp/atop/GNUmakefile

```diff
@@ -90,6 +91,9 @@ procmetrics.h:	pmgenmap.sh procmetrics.map
 acctmetrics.h:	pmgenmap.sh acctmetrics.map
 	$(PMGENMAP) acctmetrics.map > acctmetrics.h
 
+netprocmetrics.h:	pmgenmap.sh netprocmetrics.map
+	$(PMGENMAP) netprocmetrics.map > netprocmetrics.h
+
```

Comment:
Just a heads up - you may have some code here affected by the 'const' (metric names) changes from yesterday - affects output from pmgenmap too.

Reply:
thx for the heads up!
I'll do a rebase before the final PR